library/src/main/java/io/ona/kujaku/views/KujakuMapView.java

```diff
@@ -519,8 +510,6 @@ public void showCurrentLocationBtn(boolean isVisible) {
     public void focusOnUserLocation(boolean focusOnMyLocation) {
         if (focusOnMyLocation) {
             isMapScrolled = false;
```

Comment:
This has been removed

```java
changeTargetIcon(R.drawable.ic_my_location_focused);
```

Kindly add your own implementation of the same feature. Look at Google maps or OpenMapKit for illustration of the UX behind it

Reply:
Added new icon.